cleanlab/classification.py

```diff
@@ -839,6 +839,12 @@ def find_label_issues(
                 pred_probs=pred_probs,
                 thresholds=thresholds,
             )
+        # Add confident joint to find label issue args if it is not previously specified
```

Comment:
Isn't this being done too early?  Doesn't the block of code below on "if pulearning" modify self.confident_joint?

To be safe I would only do this addition right before `find_label_issues_kwargs` is actually passed into `filter.find_label_issues`